packages/backend/src/apps/postman/common/parameters-helper.ts

```diff
@@ -0,0 +1,7 @@
+import Flow from '@/models/flow'
+
+export async function getDefaultReplyTo(flowId: string): Promise<string> {
+  const flow = await Flow.query().findById(flowId).throwIfNotFound()
+  const user = await flow.$relatedQuery('user')
+  return user.email
```

Comment:
inline to save 1 query
```suggestion
const flow = await Flow.query()
    .findById(flowId)
    .withGraphFetched({ user: true })
    .throwIfNotFound()
return flow.user.email
```